src/genericPluginRegistry/plugins/NFTManager.json

```diff
@@ -30,7 +30,8 @@
           {
             "label": "Data",
             "name": "data",
-            "placeholder": ""
+            "placeholder": "Arbitrary data to transfer (0x0)",
+            "defaultValue": "0x0"
```

Comment:
It would be better if the default value were "" instead of "0x0"

Reply:
As discussed with Oren - decided to leave like this.